pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -336,6 +336,8 @@ type Config struct {
 	WorkloadMemoryDefault          *int64         `json:"workloadMemoryDefault" name:"workload-memory-default" quantity:"true" usage:"Set the default memory for acorn workloads. Accepts binary suffixes (Ki, Mi, Gi, etc) and \".\" and \"_\" seperators (default 0)" short:"m"`
 	WorkloadMemoryMaximum          *int64         `json:"workloadMemoryMaximum" name:"workload-memory-maximum" quantity:"true" usage:"Set the maximum memory for acorn workloads. Accepts binary suffixes (Ki, Mi, Gi, etc) and \".\" and \"_\" seperators (default 0)"`
 	UseCustomCABundle              *bool          `json:"useCustomCABundle" name:"use-custom-ca-bundle" usage:"Use CA bundle for admin supplied secret for all acorn control plane components. Defaults to false."`
+	AnnotationsPropagation         []string       `json:"annotationsPropagation" name:"annotations-propagation" usage:"The list of keys of annotations to propagate from acorn project to app namespaces"`
```

Comment:
Can we change this to `PropagateNamespaceLabels` and `...Annotations`.  Also the struct key and json field should be plural but the CLI arg singular, follow ClusterDomains as an example (tldr; you need "name" annotation)

Reply:
in slack, i suggested `PropagateProjectLabels`, instead of the word namespace. does that make sense @ibuildthecloud ?

Reply:
The reason that I didn't want "project" was that it's not actually the project labels, but labels on the namespace for the project.  Confusing.... But this points out we have an issue in that these labels should not go on the v1.Project object and the use shouldn't be able to change them.

Can we go with PropagateProjectNamespaceLabels then?